app/Services/UserService.php

```diff
@@ -56,4 +56,14 @@ public function update(int $id, array $inputs = []): array
 
         return $data;
     }
+
+    public function changeStatus(object $user, array $inputs = [])
+    {
+        $user->update($inputs);
+        $data = [
+            'message' => __('message.changeStatusSuccess', ['status' => $inputs['status']]),
```

Comment:
@dhrumin-7span Please add this message in the entity file

Reply:
Now, This message is from the entity file.